### DISCUSSION TITLE
exist there something Canvas like C# element?

### DISCUSSION THREAD

**Discussion by MatejMagat305** [DioxusLabs/dioxus#874] [Q&A] [answered]
hi, i m new in rust and this library, is there something like canvas in C# or javafx to paint rectangles, circles, ...?

**Comment by ealmloff** [comment, marked as answer]
You can use the canvas with web-sys to paint rectangles: https://github.com/rustwasm/wasm-bindgen/tree/main/examples/canvas. Or you could use SVGs to create shapes with elements within Dioxus: https://github.com/DioxusLabs/dioxus/blob/c5085496db057c81d859384e175d5bc636d0aaf9/examples/svg.rs
